README.md

```diff
@@ -1,6 +1,6 @@
 # Brightness Controller
 
-This is version 2 of Brightness Controller. Please note that it is compatible with python2 only. It supports an arbitrary number of displays!
+This is version 3 of Brightness Controller, ported to Python 3 and PySide 2. It supports an arbitrary number of displays!
```

Comment:
@LordAmit Does this support an arbitrary no. of displays?

Reply:
If I remember correctly, yes. It lists all your devices under primary and secondary so that you can control them as you want.